app/assets/stylesheets/scaffold.css

```diff
@@ -45,3 +45,7 @@ div.actions {
 label {
   display: block;
 }
+
+label.required:after {
+  content:"*" 
```

Comment:
You could add the following lines to display the label's asterisks exactly the same way as the asterisk in the explanatory footnote:

```css
  vertical-align: 0.5em;
  font-size: 0.75em;
  font-weight: normal;
```
(values taken from [this weird website](http://www.java2s.com/Code/HTMLCSS/Tags/fontsizeforSubscriptandSuperscript.htm))

Results:
<img width="232" alt="bildschirmfoto 2018-01-10 um 22 27 59" src="https://user-images.githubusercontent.com/807665/34796731-b84342fe-f656-11e7-8775-d6a7747f4fce.png">

![bildschirmfoto 2018-01-10 um 22 38 56](https://user-images.githubusercontent.com/807665/34796874-4bdac2ee-f657-11e7-9c0a-30e8cfde2240.png)
![bildschirmfoto 2018-01-10 um 22 38 44](https://user-images.githubusercontent.com/807665/34796875-4bf55d7a-f657-11e7-8d1e-b1cef00eb83c.png)